utils/sv_bmesh_utils.py

```diff
@@ -912,3 +913,186 @@ def recalc_normals(verts, edges, faces, loop=False):
         verts, edges, faces = pydata_from_bmesh(bm)
         bm.free()
         return verts, edges, faces
+
+def calc_center_mass_bmesh(center_mode, vertices_I, edges_I, faces_I, mass_of_vertices_I=[1], density_I=[1], skip_test_volume_are_closed=False, quad_mode="BEAUTY", ngon_mode="BEAUTY"):
```

Comment:
In python, it's bad idea to use potentially mutable values as default values for arguments. Basically you have to do def calc_center_mass_bmesh(mass_of_vertices_I = None), and then if mass_of_vertices_I is None: mass_of_vertices = [1].

Also, why _I ?

Reply:
Fix [1] to None.
_I - I use it in “for” before. It show all data connect to one object (I-th object).

Reply:
when you read this method, you shouldn't have to remember that it is called in a loop :) In another place it can be called without a loop. So, while _I names are okay in context of loop, as names of method parameter this does not make sense, imho.

Reply:
I see. Fix it. )